keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/VirtualMachineOptions.kt

```diff
@@ -0,0 +1,11 @@
+package com.netflix.spinnaker.keel.api.artifacts
+
+import com.netflix.spinnaker.keel.api.artifacts.BaseLabel.RELEASE
+import com.netflix.spinnaker.keel.api.artifacts.StoreType.EBS
+
+data class VirtualMachineOptions(
+  val baseLabel: BaseLabel = RELEASE,
+  val baseOs: String,
+  val regions: Set<String>,
+  val storeType: StoreType = EBS
+)
```

Comment:
Details needed for bake config on a `DebianArtifact`. As @erikmunson pointed out ideally we want to get rid of `regions` here and just work it out based on the resources we are deploying.

Reply:
Thoughts for a future PR - It'd be neat if keel could determine across all resources/envs every combination of (artifact, baseLabel, baseOs, storeType, regions) and enforce/trigger breaks for the superset of regions. That, or we should eliminate the requirement that image names match across all regions.

If resource in app1 uses artifact `fnord` in regions A, B and a resource in app2 uses artifact `fnord` in regions B, C, whichever gets evaluated second is going to have a hard time since the image created in region A or C will differ from B.

Reply:
I was hoping we'd evaluate all regions in the entire delivery config and just bake 'em all in one go

Reply:
There may still be times where different delivery configs for different apps reference the same artifact with a different set of regions. Consider how many spinnaker@netflix services used to have a "spinredis" package deployed amongst them, so I don't think sharing artifacts across delivery-configs will necessarily be a rare corner case.